Alicerce/Sources/Logging/Log+ConsoleLogDestination.swift

```diff
@@ -10,8 +10,8 @@ import Foundation
 
 public extension Log {
 
-    public class ConsoleLogDestination: LogDestination
-    {
+    public class ConsoleLogDestination: LogDestination {
+        
```

Comment:
This it's already turned on (option *Automatically trim trailing whitespace* as well as *Including white space only lines*, set to checked, right)?